components/SearchForm.js

```diff
@@ -39,6 +47,14 @@ const SearchButton = styled(Flex)`
   }
 `;
 
+const ClearFilterButton = styled(Flex)`
+  && {
+    appearance: none;
+    background-color: transparent;
+    border: none;
+  }
+`;
```

Comment:
- Why not use a button directly? It seems that we're not using flex capabilities below
- I assume it was done for consistency with the code above, but I don't think these `&&` are necessary here (nor in the code you took inspiration from).
- Since they're fixed, we can directly set the other CSS attributes here

```suggestion
const ClearFilterButton = styled.button`
  appearance: none;
  background-color: transparent;
  border: none;
  margin-left: 8px;
  padding: 4px;
`;
```